pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dto/TrainerRenameDTO.java

```diff
@@ -0,0 +1,47 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.dto;
+
+import static cz.muni.fi.pa165.pokemon.league.participation.manager.dto.TrainerDTOConstants.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * DTO class for change Trainer name and surname.
+ *
+ * @author Jiří Medveď 38451
+ */
+public class TrainerRenameDTO {

```

Comment:
Missing equals, hashCode and toString methods, auto generated with all properties is completely enough.